packages/app/src/explorer/Explorer.tsx

```diff
@@ -18,9 +18,13 @@ function Explorer(props: Props) {
   const ref = useRef<HTMLDivElement>(null);
   const btnRef = useRef<HTMLButtonElement>(null);
 
+  const isRootSelected = selectedPath === '/';
+
   useEffect(() => {
-    btnRef.current?.focus();
-  }, [btnRef]);
+    if (isRootSelected) {
+      btnRef?.current?.focus();
+    }
+  }, [btnRef, isRootSelected]);
```

Comment:
Same as before but for the `/` path since the file button does not use `EntityItem`